test/scenarios.py

```diff
@@ -3821,6 +3821,18 @@
                     ),
                     action.WaitRacy(
                         [
+                            event.Start(
+                                0,
+                                FILES[
+                                    "thisisaverylongfilenameusingonlylowercaselettersandnumbersanditcontainshugestringofnumbers01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234561234567891234567891234567890123456789012345678901234567890123456.txt"
+                                ].id,
+                            ),
+                            event.Start(
+                                1,
+                                FILES[
+                                    "thisisaverylongfilenameusingonlylowercaselettersandnumbersanditcontainshugestringofnumbers01234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234561234567891234567891234567890123456789012345678901234567890123456.txt"
+                                ].id,
```

Comment:
Could you explain a bit more why this PR triggered a change in this testcase?

Reply:
I'm not sure, probably the test was failing on dev already and we simply were ignoring this